conf/jasminegraph-server.properties

```diff
@@ -25,6 +25,8 @@ org.jasminegraph.server.mode.isdistributed=false
 org.jasminegraph.server.host=localhost
 #org.jasminegraph.server.runtime.location is the location where termprary files, working directories of
 org.jasminegraph.server.nworkers=2
+#org.jasminegraph.server.npartitions is the number of partitions into which the graph shpuld be partitioned
```

Comment:
Please correct spelling the mistake "which the graph shpuld"

Reply:
Corrected